signal/store.go

```diff
@@ -18,6 +23,63 @@ func NewStore(ctx context.Context, db *sql.DB) *Store {
 	return &Store{db: db}
```

Comment:
This is more of a nitpick but we could pass in the db transactions as a `gadb.DBTX` parameter instead of storing the db as a field in the store obj. It'll be more explicit and readable this way but just less convenient i suppose.

Reply:
Looks good aside from that though!

Reply:
The HTTP endpoint currently doesn't have direct access to the database. Do you think we should pass it into the handler config? That would work, but it goes against what the other HTTP handlers currently do.

Reply:
for reference, here are the Store calls in the http handler:
https://github.com/target/goalert/blob/521869d33c3ac92ed38fae4bad283a33e5088c94/signalapi/notify.go#L91
https://github.com/target/goalert/blob/521869d33c3ac92ed38fae4bad283a33e5088c94/signalapi/rule.go#L16

Reply:
I think it's fine to add db to the handler config and setting it to `app.db` in inithttp.go